### ISSUE TITLE
[Feature Request] Multiple baselines for the same color

### ISSUE THREAD

**Issue by tylerwiegand** [javisperez/tailwindcolorshades#10]
The ability to input multiple colors in different parts of the array -- for instance, my designer gives me 3 colors to work with, and I would plug them into different places on the "brightness spectrum," but I want all the in-betweens too, because tailwind!

Now I have to use some nasty thing like [this](https://meyerweb.com/eric/tools/color-blend) instead of your cool tool so I can "blend."

Love the tool though!

**Comment by hacknug** [comment]
Have you tried using [`chroma-js`](https://github.com/gka/chroma.js/) to generate your color palette?

**Comment by tylerwiegand** [comment]
well holy balls thatd do it pretty nicely wouldnt it?

**Comment by mikemand** [comment]
Do you have an example of what kind of input you would want to provide and what kind of output you are expecting?

**Comment by tylerwiegand** [comment]
Yes absolutely! So for instance, in the picture of the current tool, each of the shades, when clicked, would be editable. Then, if you enter a different value into one of them, the shades around them would change according to the difference in hue/saturation/etc between those two colors but the available colors still extend to the normal 100-900 range.

So for instance, you enter color-x which becomes flower-500. But you wanted flower-200 to be a bit less blue and to wash out a little more or something, so you pick what you would want for flower-200 and then flower-100, 300, and 400 all change accordingly. Then, depending on the "direction" of that change, it also would effect 600-900 as well.

I personally don't know a ton about how color differences can be calculated, but if that chrome-js tool exists what I'm proposing sounds possible to me...

![image](https://user-images.githubusercontent.com/11890164/70662189-98698100-1c1a-11ea-937b-2ac2148bcfeb.png)

**Comment by javisperez** [comment]
sorry for the late response, its been a year now 😬. I'm thinking on this, I want to add it, just need a few days to do it but this makes a lot of sense to me. Will keep this open until is finally done.

**Comment by javisperez** [comment]
Alright, so just like i mention in #18 i added now an option for pin and customize colors in the palette. Is not 100% the same but at least puts us in the right direction. Color 500 is used as the base and updating that is a bit tougher than expect so that one must remain as selected, but it should be good enough to unblock this issue.

In the meantime ill close this and we can always reopen a new one if we need the feature to be tweak.

Please hard refresh your browser before testing! i'm seeing a crazy browser cache from github pages.
